src/components/shared/customFooter.js

```diff
@@ -28,9 +28,9 @@ const DefaultFooter = ({logos, text, widgets}) => (
       <div className="col-md-3 content-area">
```

Comment:
Remove unneeded call to `contentExists` in line 27

Reply:
Removed contentExists